core/src/main/java/com/novoda/noplayer/internal/exoplayer/MediaCodecVideoRendererWithSimplifiedDrmRequirement.java

```diff
@@ -70,27 +78,38 @@
             boolean requiresSecureDecoder,
             boolean requiresTunnelingDecoder)
             throws MediaCodecUtil.DecoderQueryException {
-        List<MediaCodecInfo> decoderInfos =
-                mediaCodecSelector.getDecoderInfos(
-                        format.sampleMimeType, requiresSecureDecoder, requiresTunnelingDecoder);
+        List<MediaCodecInfo> decoderInfos = mediaCodecSelector.getDecoderInfos(
```

Comment:
no change in this class, just some refactoring so that we can understand the code better